File: Data Structures & Algorithms/kth-largest-element-in-an-array/submission-0.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        PriorityQueue<Integer> maxHeap = new PriorityQueue<>(Collections.reverseOrder());
        for (int i = 0; i < nums.length; i++) {
            maxHeap.add(nums[i]);
        }

        for (int j = 0; j < k -1; j++) {
            maxHeap.poll();
        }


        return maxHeap.peek();
    }
}

File: Data Structures & Algorithms/climbing-stairs/submission-0.py
class Solution:
    def climbStairs(self, n: int) -> int:
        def dp(n):
            if n == 2:
                return 2
            if n == 1:
                return 1

            else:
                return dp(n-1) + dp(n-2)
        return dp(n)        
